spec/std/hash_spec.cr

```diff
@@ -279,12 +279,12 @@ describe "Hash" do
   end
 
   describe "to_s" do
-    assert { {1 => 2, 3 => 4}.to_s.should eq("{1 => 2, 3 => 4}") }
+    assert { {1 => 2, 3 => 4}.to_s.should eq("{1=>2, 3=>4}") }
```

Comment:
I'm not sure about this, I would prefer if by default the generated representation is valid and properly formatted Crystal code, so paste the output around doesn't trigger formatter differences?